src/main/java/build/buildfarm/admin/Admin.java

```diff
@@ -29,4 +29,6 @@ void scaleCluster(
       Integer maxHosts,
       Integer targetHosts,
       Integer targetReservedHostsPercent);
+
+  void disableHostProtection(String scaleGroupName, String hostId);
```

Comment:
can you remind me how host protection gets enabled?  

I think all the instances have host protection automatically by default from aws?  And our only use case is to turn it off when shutting down a worker.  

Is host protection applied to schedulers or only workers?

Reply:
Yes, all worker hosts will have the protection on default when we roll out the graceful shutdown. I'm not super sure the are the default on was.

Scale in protection is an aws instance feature, so it also applies to scheduler instances. But i don't remember we have any plan to turn it on for schedulers.